### PR TITLE
Restrict minimum interval of password reset

### PR DESCRIPTION
**Pull request by niauah**

### CODE REVIEW

File: password_policies/password_validation.py [hunk 1]
```diff
@@ -56,7 +59,18 @@ def get_help_text(self):
...
-class RepeatedValidator:
+class CreatePasswordRecordMixin:
+
+    def password_changed(self, password, user=None):
+        # In case there is no user, this is not applicable.
+        if user is None:
+            return None
+
+        hashed_password = make_password(password)
+        user.password_records.create(password=hashed_password)
```
Comment (sih4sing5hong5) [inline password_policies/password_validation.py:70]: 敢是`RepeatedValidator` kah `MinimumResetIntervalValidator`攏會create record?
按呢敢會create 2條出來？

**Comment by niauah** [comment]
[Reusable apps and **`AUTH_USER_MODEL`**](https://docs.djangoproject.com/en/4.1/topics/auth/customizing/#reusable-apps-and-auth-user-model)

> Reusable apps shouldn’t implement a custom user model. A project may use many apps, and two reusable apps that implemented a custom user model couldn’t be used together.

**Comment by sih4sing5hong5** [comment]
signal:

- https://docs.djangoproject.com/en/4.1/topics/signals/
- https://docs.djangoproject.com/en/4.1/ref/signals/#pre-save

File: password_policies/signals.py [hunk 2]
```diff
@@ -4,5 +4,5 @@
 
 
 @receiver(post_save, sender=get_user_model())
-def my_callback(sender, **kwargs):
-    print('Signal connected')
+def my_callback(sender, instance, **kwargs):
```
Comment (sih4sing5hong5) [inline password_policies/signals.py:7]: 著，`my_callback`名改改--leh就差不多--ah

Comment (niauah) [reply password_policies/signals.py]: 732b9c5

**Review by sih4sing5hong5 — APPROVED** [review]

**Pull request merged**